### PR TITLE
EUI-4170: Roles and access > Delete exclusion > "Check your answers" page

### PR DESCRIPTION
**Pull request by Daniel-Lam**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4170

### Change description ###
Implement display-only User Story.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/role-access/containers/delete-exclusion/delete-exclusion.component.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  constructor(private readonly route: ActivatedRoute) {}
+
+  public ngOnInit(): void {
+    // Get the role exclusions from the route, which will have been put there by the resolver.
+    const roleExclusions = this.route.snapshot.data.roleExclusions;
+    this.populateAnswers(roleExclusions);
+  }
+
+  public onEventTrigger(event: ExclusionNavigationEvent) {
```
Comment (connorpgpmcelroy) [inline src/role-access/containers/delete-exclusion/delete-exclusion.component.ts:29]: Method has no void return type

File: src/role-access/containers/delete-exclusion/delete-exclusion.component.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    this.eventTrigger.emit(event);
+  }
+
+  private populateAnswers(exclusions: any[]) {
```
Comment (connorpgpmcelroy) [inline src/role-access/containers/delete-exclusion/delete-exclusion.component.ts:33]: Could any[] change to be Exclusions or similar? Void return type for this as well?

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Just two minor comments

**Review by rayliangatos — APPROVED** [review]

File: src/role-access/containers/delete-exclusion/delete-exclusion.component.ts (hunk 1 above)
Comment (Daniel-Lam) [reply src/role-access/containers/delete-exclusion/delete-exclusion.component.ts:29]: @connorpgpmcelroy - good catch! Will add `void`

File: src/role-access/containers/delete-exclusion/delete-exclusion.component.ts (hunk 2 above)
Comment (Daniel-Lam) [reply src/role-access/containers/delete-exclusion/delete-exclusion.component.ts:33]: @connorpgpmcelroy - you're right... I'll fix both of these

**Review by connorpgpmcelroy — APPROVED** [review]
Have checked throughout and all looks good

**Pull request merged**